src/Mnemonic.h

```diff
@@ -13,10 +13,9 @@ namespace TW {
 /// BIP39 Mnemonic Sentence handling.
 class Mnemonic {
 public:
-    static constexpr size_t MaxWords = 24;
-    static constexpr size_t MinWords = 12;
-    static constexpr size_t BitsPerWord = 11; // each word encodes this many bits (there are 2^11=2048 different words)
-    static constexpr size_t MaxWordLength = 9; // Length of longest mnemonic word
+    static constexpr int MaxWords = 24;
```

Comment:
should we hardcode this into bip39.h as we did with `BIP39_MAX_WORDS` ?

Reply:
I've considered, but I try to minimize downstream changes in trezor-crypto, I don't want to add constant there which is not directly used inside the lib.